feature/bgp/policybase/otg_tests/link_bandwidth_test/README.md

```diff
@@ -0,0 +1,149 @@
+    * Create an ext-community-set named 'linkbw_2G' with members as follows:
+      * ext-community-member = [ "link-bandwidth:100:2G" ]
+    * Create an community-set named 'regex_match_as100' with members as follows:
+      * ext-community-member = [ "^100:.*$" ]
+    * Create an community-set named 'regex_nomatch_as100' with members as follows:
+      * ext-community-member = [ "^100:.*$" ]
+    * Create an ext-community-set named 'linkbw_any' with members as follows:
+      * ext-community-member = [ "^link-bandwidth:.*:.*$" ]
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'set_linkbw_0' with the following `statements`
+    * statement[name='zero_linkbw']/
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_0']
+      * actions/bgp-actions/set-ext-community/config/options = ADD
+      * actions/bgp-actions/set-community/config/method = REFERENCE
+      * actions/config/policy-result = NEXT_STATEMENT
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'match_100_set_linkbw_1M' with the following `statements`
+    * statement[name='1-megabit-match']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'regex_match_as100'
+      * conditions/bgp-conditions/match-ext-community-set/config/match-set-options = INVERT
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_1M']
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'nomatch_100_set_linkbw_2G' with the following `statements`
+    * statement[name='2-gigabit-match']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'regex_nomatch_as100'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_2G']
+    * statement[name='accept_all_routes']/
+      * actions/config/policy-result = ACCEPT_ROUTE
+
+  * Create a `/routing-policy/policy-definitions/policy-definition/policy-definition`
+    named 'del_linkbw' with the following `statements`
+    * statement[name='del_linkbw']/
+      * conditions/bgp-conditions/match-ext-community-set/config/community-set = 'linkbw_any'
+      * actions/bgp-actions/set-ext-community/reference/config/ext-community-set-refs =
+          /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set[name='linkbw_any']
```

Comment:
where we state that this community is to be DELETED. we need `option` leaf

Reply:
Thanks!  Done